DashWallet/Sources/Models/CrowdNode/CrowdNode+UserDefaults.swift

```diff
@@ -32,19 +32,25 @@ private let kOnlineInfoShown = "crowdNodeOnlineInfoShownKey"
 private let kSignedEmailMessageId = "crowdNodeSignedEmailMessageId"
 private let kShouldShowConfirmedNotification = "shouldShowConfirmedNotification"
 
+// MARK: - CrowdNodeDefaults
+
 class CrowdNodeDefaults {
     public static let shared: CrowdNodeDefaults = .init()
-    
-    var crowdNodeAccountAddress: String? {
-        get { UserDefaults.standard.value(forKey: kCrowdNodeAccountAddress) as? String }
-        set(value) { UserDefaults.standard.set(value, forKey: kCrowdNodeAccountAddress) }
+
+    private var _accountAddress: String? = nil
+    var accountAddress: String? {
+        get { _accountAddress ?? UserDefaults.standard.value(forKey: kCrowdNodeAccountAddress) as? String }
+        set(value) {
+            _accountAddress = value
+            UserDefaults.standard.set(value, forKey: kCrowdNodeAccountAddress)
+        }
```

Comment:
Initially, I left out caching from `crowdNodeAccountAddress` to avoid issues with restoring and synchronizing the addresses fields.
However, if we leave the address field in here only and keep the restoring logic as it is, then it's fine.